x-pack/plugins/security_solution/public/hosts/containers/kpi_hosts/risky_hosts/index.tsx

```diff
@@ -68,14 +70,30 @@ const getRiskyHostsWithOptionalSignal = withOptionalSignal(getRiskyHostsComplete
 const useRiskyHostsComplete = () => useObservable(getRiskyHostsWithOptionalSignal);
 
 interface UseRiskyHostProps {
-  filterQuery?: string;
+  filterQuery?: string | ESTermQuery;
   from: string;
   to: string;
-  skip: boolean;
+  skip?: boolean;
+}
+export type SeverityCount = {
+  [k in HostRiskSeverity]: number;
+};
+
+interface RiskScoreKpi {
+  // TODO: Pablo what is this error/isModuleDisabled value supposed to be??
+  error: unknown;
```

Comment:
https://github.com/elastic/kibana/pull/122980/files#r790756976

help me figure out these types and i'll switch the other one over

Reply:
isModuleDisabled is a boolean

Reply:
You can make it a boolean by double `!` error:
```
const isModuleDisabled = !!error && isIndexNotFoundError(error);
```

Reply:
Error is defined as `unknow` here:
`node_modules/@types/kbn__securitysolution-hook-utils/index.d.ts`

Reply:
perfect thanks!